.eslintrc

```diff
@@ -86,6 +86,12 @@
       "rules": {
         "no-console": "error"
       }
+    },
+    {
+      "files": ["www/api/**", "www/og-image/**"],
+      "rules": {
+        "react/no-unknown-property": "off"
+      }
```

Comment:
I'd prefer if this was an eslint comment at top of the files

Otherwise, if we remove the files, this will be a dangling check
```suggestion
```